state/postgres_statement.go

```diff
@@ -104,3 +104,11 @@ func (stmt DiffedPostgresStatementStats) Add(other DiffedPostgresStatementStats)
 		BlkWriteTime:      stmt.BlkWriteTime + other.BlkWriteTime,
 	}
 }
+
+type QueryIdentity struct {
+	QueryID     int64
+	Fingerprint uint64
+	LastSeen    time.Time
```

Comment:
I was surprised to see that the size of time.Time is 24 bytes (https://stackoverflow.com/questions/56963226/how-many-bytes-are-in-a-golang-time-object), making this pretty expensive - maybe we can instead keep a uint64 as a unix timestamp field?

Reply:
Good point, will update.